.github/workflows/test-pygenesys.yml

```diff
@@ -0,0 +1,66 @@
+# Preamble
+name: Test Pygenesys
+
+on:
+  push:
+    paths:
+      - 'pygenesys/**'
+      - 'pygenesys/tests/**'
+  pull_request:
+  # enable worflow to be run manually
+  workflow_dispatch:
+
+env:
+  CACHE_NUMBER: 0 #change to manually reset cache
+
+jobs:
+  test-pygenesys:
+    runs-on: ubuntu-latest
```

Comment:
is it possible to also run-on windows latest?

Reply:
It's very possible, would need to slightly readjust some of the syntax, but very easy to do.